src/domain/mongoose_domain_utils.erl

```diff
@@ -0,0 +1,6 @@
+-module(mongoose_domain_utils).
+-export([halt_node/1]).
+
+halt_node(ExitText) ->
```

Comment:
Just a side note, no need to change anything:

we should think about one common MIM helper module for such functionality